token/program-2022/src/extension/default_account_state/instruction.rs

```diff
@@ -0,0 +1,115 @@
+use {
+    crate::{
+        check_program_account, error::TokenError, instruction::TokenInstruction,
+        state::AccountState,
+    },
+    num_enum::{IntoPrimitive, TryFromPrimitive},
+    solana_program::{
+        instruction::{AccountMeta, Instruction},
+        program_error::ProgramError,
+        pubkey::Pubkey,
+    },
+    std::convert::TryFrom,
+};
+
+/// Default Account State extension instructions
+#[derive(Clone, Copy, Debug, PartialEq, IntoPrimitive, TryFromPrimitive)]
+#[repr(u8)]
+pub enum DefaultAccountStateInstruction {
```

Comment:
These two instructions could use documentation about the required instruction data (an `AccountState`)

Reply:
Oh true. I missed your pattern for this.